data/Konpal-Sharma.json

```
@@ -0,0 +1,12 @@
+{
+  "name: "Konpal Sharma",
+  "type": "personal",
+  "bio": "CS student"
```

JSON needs to be corrected: 
```suggestion
	"name": "Konpal Sharma ",
	"type": "personal",
	"bio": "CS student",
```